agent/temboardagent/cli/register.py

```diff
@@ -158,7 +158,7 @@ def main(self, args):
                 'groups': groups
             })
             response.raise_for_status()
-            dashboard_url = ui_url_raw + '/server/%s/%s/dashboard' % (
+            dashboard_url = ui_url_raw + '/server/{}/{}/dashboard'.format(
```

Comment:
I think we can use f-string here as well.

Reply:
```
            dashboard_url = (
                f"{ui_url_raw}/server/"
                f"{args.host}/{self.app.config.temboard.port}/dashboard")
```
like this?

Reply:
we choose to keep format and to include ui_url_raw in it.